doc/plugins.dox

```diff
@@ -807,17 +806,17 @@ Consider the 5 basic cases:
 1) A candidate comes with a suitable file extension but is not a workable plugin file at all. For
 example, your proxy supports plugins written in a shell script (.sh) but the shebang of that script
 points to an incompatible shell (or even lacks a shebang). You should check for this in
-GeanyProxyFuncs::probe() and return @ref PROXY_IGNORED which hides that script from the Plugin
+GeanyProxyFuncs::probe() and return @ref GEANY_PROXY_IGNORE which hides that script from the Plugin
 Manager and allows other enabled proxy plugins to pick it up. GeanyProxyFuncs::probe() returning
-@ref PROXY_IGNORED is an indication that the candidate is meant for another proxy, or the user
+@ref GEANY_PROXY_IGNORE is an indication that the candidate is meant for another proxy, or the user
```

Comment:
What? It's [still in the enum](https://github.com/geany/geany/blob/master/src/plugindata.h#L356) and [documented there](https://github.com/geany/geany/blob/master/src/plugindata.h#L356) and in the [enum doc-comment](https://github.com/geany/geany/blob/master/src/plugindata.h#L344). I don't understand.